swagger.yml

```diff
@@ -11,7 +11,7 @@ info:
   license:
     name: "MIT"
     url: "https://opensource.org/licenses/MIT"
-host: "localhost:38199"
+host: "localhost:39413"
```

Comment:
we should add `.host("N/A")` to ApiDocumentation

Reply:
Done.

Reply:
you need to re-run the build too I'm afraid, so that this changes to N/A